wallet/how-to/register-token.md

```diff
@@ -46,8 +66,89 @@ try {
 }
 ```
 
+:::note
+If the chain ID of your token doesn't match the user's network, they can get unexpected results.
+We recommend [detecting the user's network chain ID](../get-started/detect-network.md) and
+[prompting them to switch chains](/wallet/reference/wallet_switchethereumchain), if necessary.
+:::
+
 For more examples, the following are live web dapps that let you enter your token details and share
 them using a simple web link:
 
 - [WatchToken](https://vittominacori.github.io/watch-token/create/)
 - [Add Token dapp](https://metamask.github.io/Add-Token/#edit)
+
+## Display NFTs
```

Comment:
Nit: should we break this into 2 topics? Display a single NFT and Display multiple NFTs